mpv.spec

```diff
@@ -95,15 +95,16 @@ waf configure \
     --disable-build-date \
     --enable-libmpv-shared \
     --enable-gpl3 \
-    --enable-sdl2
+    --enable-sdl2 \
+    --enable-encoding
 
-waf build --verbose %{?_smp_mflags}
+waf build %{?_smp_mflags}
 
 %install
-waf --destdir=%{buildroot} install %{?_smp_mflags}
+waf install --destdir=%{buildroot}
 
 desktop-file-install etc/mpv.desktop
-install -Dpm 644 etc/%{name}-symbolic.svg %{buildroot}%{_datadir}/icons/hicolor/symbolic/apps/%{name}-symbolic.svg
+##install -Dpm 644 etc/%{name}-symbolic.svg %{buildroot}%{_datadir}/icons/hicolor/symbolic/apps/%{name}-symbolic.svg
```

Comment:
Hi, this icon is now installed by default.
I missed this line, then next time I delete this line